src/screens/DetailedViewScreen.tsx

```diff
@@ -65,16 +69,14 @@ const DetailedViewScreen: React.FC<PlaceProps> = ({ setPage, placeID }: PlacePro
 					</BodyText>
 					<Button
 						style={styles.submitButton}
-						onPress={() => {
-							//placeholder
-						}}
+						onPress={() => goToSubmitRatingScreen()}
 						//onPress={() => setPage("login")} need to update this after Eleni's PR is merged w/ Andrew's updates to nav
 						accessibilityLabel="Submit an accessibility rating"
 						text="Submit a Rating"
 					/>
 					<Button
 						style={styles.homeBtn}
-						onPress={() => setPage("mapScreen")} //need to update this after Eleni's PR is merged w/ Andrew's updates to nav
+						onPress={() => goToMapScreen()} //need to update this after Eleni's PR is merged w/ Andrew's updates to nav
```

Comment:
should this comment be removed?

Reply:
I made some changes in this file in #160 and I removed them there @aej11a